optimum/intel/openvino/quantization.py

```diff
@@ -1365,7 +1405,7 @@ def _quantize_torchmodel(
             model = nncf.quantize(
                 model,
                 calibration_datasets["model"],
-                subset_size=quantization_config.num_samples,
+                subset_size=quantization_config.num_samples or 128,
```

Comment:
why is this value hidden here ?

Reply:
128 was previously a default value for `num_samples` argument in `OVQuantizationConfig`. In this PR I've removed it so it is now `None` by default. That's why 128 has appeared here.

Ideally, we should transition to providing arguments via `quantization_config.to_nncf_dict()` here, as it is done for OV case. But I propose to do this in a separate PR.